geoextent/__main__.py

```diff
@@ -0,0 +1,92 @@
+
+import sys, os, getopt, datetime
+import geoextent.lib.helpfunctions as hf
+import geoextent.lib.extent as extent
+import logging
+
+COMMAND = None
+
+# the capabilities of our CLI
+def usage():
+    """Provide usage instructions"""
+    return '''
+        NAME
+            geoextent.py 
+
+        Usage example
+            extract_metadata.py -b </absoulte/path/to/directory>
```

Comment:
Are really only absolute paths supported? By default, geoextent should just look at the current path, and also support relative paths starting from the directory where the CLI is called.

Reply:
So far yes. Is your point to add search in folder for files.

Reply:
My point is: let's support relative paths to where the tools is started!

With [`os.path.isabs()`](https://docs.python.org/3/library/os.path.html#os.path.isabs) you can check if a path is absolute, and if is not you can build a full path with `os.getcwd()`.

Reply:
now I got what you mean :+1:

Reply:
Done.